i18n/states.php

```diff
@@ -269,6 +269,24 @@
 		'ZG' => __( 'Zug', 'woocommerce' ),
 		'ZH' => __( 'Zürich', 'woocommerce' ),
 	),
+	'CL' => array( // Chile states.
+		'CL-AI' => __( 'Aisén del General Carlos Ibañez del Campo', 'woocommerce' ),
+		'CL-AN' => __( 'Antofagasta', 'woocommerce' ),
+		'CL-AP' => __( 'Arica y Parinacota', 'woocommerce' ),
+		'CL-AR' => __( 'La Araucanía', 'woocommerce' ),
+		'CL-AT' => __( 'Atacama', 'woocommerce' ),
+		'CL-BI' => __( 'Biobío', 'woocommerce' ),
+		'CL-CO' => __( 'Coquimbo', 'woocommerce' ),
+		'CL-LI' => __( 'Libertador General Bernardo O Higgins', 'woocommerce' ),
```

Comment:
Apologies because I thought I commented on this line at an earlier stage—but it seems like the comment wasn't saved, for some reason.

Should this be:

```suggestion
		'CL-LI' => __( 'Libertador General Bernardo O'Higgins', 'woocommerce' ),
```

[(Ref)](https://www.iso.org/obp/ui/#iso:code:3166:CL)